packages/core/src/Services/socket-general.js

```diff
@@ -97,9 +97,13 @@ const BinarySocketGeneral = (() => {
                 break;
             case 'get_settings':
                 if (response.get_settings) {
+                    const updated_settings = {
+                        ...response.get_settings,
+                        phone: response.get_settings?.phone.replace(/[^0-9+]/g, ''),
```

Comment:
why do we need this change in socket-general?

Reply:
I was thinking to update all phone numbers for account settings. Should we update it in general websocket or we could just do it only in personal-details-form 🤔

Reply:
IMO we should not manipulate the data coming from the API

Reply:
the replacement of the spaces or hyphen if available will be trimmed updating the values again in the personal details section

If the data is needed to be replaced in the DB, it should be done on BE

Reply:
IMO, we shouldnot manipulate data here. You can do the formating in personal details page.

Reply:
Will do the formatting only in personal details page @amina-deriv.